spec/realworld/edgecases_spec.rb

```diff
@@ -23,12 +23,12 @@ def rubygems_version(name, requirement)
     gemfile <<-G
       source "https://rubygems.org"
 
-      gem 'rails', '~> 3.0'
+      gem 'rails', '~> 5.0'
```

Comment:
@colby-swandale I'd be tempted to say: this spec relied on bundler 1 because rails 3 depends on bundler 1, so it's no longer a meaningful realworld spec for bundler 2 and just drop it. BUT, the description is so generic ("resolves dependencies correctly"), that I'm good with your alternative fix of changing the spec. Thanks! :+1: